.devcontainer/README.md

```diff
@@ -2,6 +2,40 @@
 
 For a comprehensive developer environnment supporting AGW c / c++ workflows (extensible with ~effort to other Magma uses) - see the instructions below.
 
+## Prerequisites and Restrictions
+
+DevContainer is a technology developed by Microsoft. Thus, it is only available for [_VS Code_](https://code.visualstudio.com/docs/remote/containers), but **not** for _IntelliJ_. However, as Github is also currently owned by Microsoft, Github provides a DevContainer integration named ["Codespaces"](https://docs.github.com/en/codespaces).
+In order to use the Github codespaces, you must be member of the Magma Github organisation.
+In any case you will need a working Docker installation, as DevContainer is a docker based technology.
+
+Prerequisites in Summary:
+* Local usage:
+  * Docker installed
+  * VS code installed
```

Comment:
Very good idea!

As a next step one could even have a look, whether a predefined workspace equivalent to the ones defined in https://github.com/magma/magma/tree/master/vscode-workspaces would be an option.
But I would prefer to have this as a separate step and PR, as this would be another topic (for me at least).